includes/Dashboard/Templates/Orders.php

```diff
@@ -85,22 +85,21 @@ public function order_main_content() {
      * @return void
      */
     function handle_order_export() {
-        if ( ! is_user_logged_in() ) {
+        if ( ! isset( $_POST['dokan_vendor_order_export_nonce'] ) || ! wp_verify_nonce( sanitize_key( wp_unslash( $_POST['dokan_vendor_order_export_nonce'] ) ), 'dokan_vendor_order_export_action' ) ) {
             return;
         }
 
-        if ( ! dokan_is_user_seller( get_current_user_id() ) ) {
+        // return if is not vendor or vendor staff
+        if ( ! dokan_is_user_seller( dokan_get_current_user_id() ) ) {
             return;
         }
 
-        $post_data = wp_unslash( $_POST );
-
-        if ( ! isset( $post_data['dokan_vendor_order_export_nonce'] ) || ! wp_verify_nonce( sanitize_key( $post_data['dokan_vendor_order_export_nonce'] ), 'dokan_vendor_order_export_action' ) ) {
+        // if current user is vendor staff and has proper permission
+        if ( current_user_can( 'vendor_staff' ) && ! current_user_can( 'dokan_export_order' ) ) {
```

Comment:
Can we add only `current_user_can( 'dokan_export_order' )`

Reply:
vai, here we are checking if the current user is a staff and has the capability dokan_export_order. Because only staff with this capability has the permission.